.eslintrc.js

```diff
@@ -3,19 +3,27 @@ module.exports = {
   env: {
     node: true,
   },
-  extends: ['plugin:vue/recommended', '@vue/airbnb'],
+  extends: ["plugin:vue/recommended", "@vue/airbnb"],
   parserOptions: {
-    parser: 'babel-eslint',
+    parser: "babel-eslint",
   },
   rules: {
-    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
-
-    semi: ['error', 'never'],
-    'max-len': 'off',
-    'linebreak-style': 'off',
-    camelcase: ['error', { properties: 'never', ignoreDestructuring: true, ignoreImports: true }],
-    'arrow-parens': ['error', 'as-needed'],
-    'vue/multiline-html-element-content-newline': 'off',
+    /* eslint quotes: ["off", "double"], curly: 2 */
```

Comment:
almost many changes are made by these changes. This really should NOT be include in this PR. 
FYI. testnet should added on `testnet` branch

Reply:
Thanks, for the call out. We forgot we had done some testing. We will remove the PR and submit a new one.